settings-boundary/HelpAndSupport.tsx

```diff
@@ -0,0 +1,278 @@
+        onPress: reportWithoutLogs
+      },
+      {
+        text: "What is this?",
+        onPress: logQuestion
+      }
+    ]
+  },
+  reportBugSuccess: {
+    title: "Received!",
+    description:
+      "Thank you for reporting this bug and helping us improve the app. We appreciate your input."
+  },
+  reportBugError: {
+    title: "Oops!",
+    description:
+      "Something went wrong while reporting this bug. Please try again."
+  },
+  compileLogError: {
+    title: "Oops!",
+    description:
+      "We're sorry, we had an error compiling logs. Sending bug report without logs.",
+    buttons: (confirmLogsCompileError: () => void) => [
+      {
+        text: "OK",
+        onPress: confirmLogsCompileError
+      }
+    ]
+  },
+  submitQuestionSuccess: {
+    title: "Received!",
+    description:
+      "Thank you for submitting your question. We’ll get back to you shortly. In the meantime, please visit our Help Center for FAQ’s and additional resources."
+  },
+  submitQuestionError: {
+    title: "Oops!",
+    description:
+      "Something went wrong while submitting your question. Please try again."
+  }
+}
+
+export type EventSettingsProps = {
+  style?: StyleProp<ViewStyle>
+  state: ReturnType<typeof useHelpAndSupportSettings>
+}
+
+export const HelpAndSupportView = ({ style, state }: EventSettingsProps) => (
+  <SettingsScrollView style={style}>
+    <HelpSectionView state={state} />
+  </SettingsScrollView>
+)
+
+export type UseHelpAndSupportSettingsEnvironment = {
+  isShowingContactSection: () => Promise<boolean>
+  compileLogs: () => Promise<string>
+  composeEmail: (email: EmailTemplate) => Promise<EmailCompositionResult>
+}
+
+export const useHelpAndSupportSettings = (
+  env: UseHelpAndSupportSettingsEnvironment
+) => {
+  const { data: isShowingContactSection } = useQuery(
+    ["isMailComposerAvailable"],
+    async () => await env.isShowingContactSection(),
+    { initialData: true }
+  )
+  const open = useOpenWeblink()
+
+  return {
+    isShowingContactSection,
+    feedbackSubmitted: async () => {
+      try {
+        const status = await env.composeEmail(
+          HELP_AND_SUPPORT_EMAILS.feedbackSubmitted
+        )
+        if (status === "success") {
+          Alert.alert(
+            HELP_AND_SUPPORT_ALERTS.submitFeedbackSuccess.title,
+            HELP_AND_SUPPORT_ALERTS.submitFeedbackSuccess.description
+          )
+        }
+      } catch {
+        Alert.alert(
+          HELP_AND_SUPPORT_ALERTS.submitFeedbackError.title,
+          HELP_AND_SUPPORT_ALERTS.submitFeedbackError.description
+        )
+      }
+    },
+    bugReported: async () => {
+      Alert.alert(
+        HELP_AND_SUPPORT_ALERTS.reportBugTapped.title,
+        HELP_AND_SUPPORT_ALERTS.reportBugTapped.description,
+        HELP_AND_SUPPORT_ALERTS.reportBugTapped.buttons(
+          async () => {
+            await env
+              .compileLogs()
+              .then(async (URI) => {
+                try {
```

Comment:
Don't really need a .then/.catch block, attempt try/catch block for less code clutter